src/msac.rs

```diff
@@ -175,6 +175,10 @@ fn ctx_norm(s: &mut MsacContext, dif: ec_win, rng: libc::c_uint) {
     }
 }
 
+#[cfg_attr(
+    all(feature = "asm", any(target_arch = "x86_64", target_arch = "aarch64")),
+    allow(dead_code)
+)]
```

Comment:
Why use `cfg_attr` here to `allow(dead_code)` instead of just `cfg`ing the entire function out when it's unused? Same for the other functions in this file where you did `cfg_attr`.

Reply:
If I `#[cfg]` the whole thing out it doesn't type check unless it's specifically enabled and rust-analyzer stops working, so it's much nicer to have it enabled and just dead code eliminated.